src/cluster.c

```diff
@@ -1300,12 +1300,39 @@ static void appendClusterNodeString(RedisModuleString *ret, char node_id[41], No
     RedisModule_FreeString(NULL, str);
 }
 
+static void appendSpecialClusterNodeString(RedisModuleString *ret, unsigned int j, SlotRangeType type, ShardGroupNode *node)
+{
+    size_t len;
+    const char *temp;
+    char *direction;
+
+    switch (type) {
+        case SLOTRANGE_TYPE_IMPORTING:
+            direction = "<";
+            break;
+        case SLOTRANGE_TYPE_MIGRATING:
+            direction = ">";
+            break;
+        default:
+            return;
+    }
+
+    RedisModuleString *str = RedisModule_CreateStringPrintf(NULL, "[%u-%s-%.*s]\r\n", j, direction, 40, node->node_id);
```

Comment:
If I understand correctly, separator should be space, not `\r\n`: https://github.com/redis/redis/blob/36abc0fa8f50a0c5bbe8b5ddcca657d6ada94363/src/cluster.c#L4775

Also, looks like whole `cluster node` format separated with `\n`, not `\r\n`: 
https://github.com/redis/redis/blob/36abc0fa8f50a0c5bbe8b5ddcca657d6ada94363/src/cluster.c#L4858

I think, the best way is to get raw output from Redis and mimic it byte by byte (seems like documentation might be misleading).
Probably, we should do it for other commands(cluster slots, cluster shards) as well, not to miss anything.